examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -90,15 +91,16 @@
                 "name": "jenkins",
                 "image": " ",
                 "readinessProbe": {
-                  "timeoutSeconds": 3,
+                  "timeoutSeconds": 240,
                   "initialDelaySeconds": 3,
                   "httpGet": {
                     "path": "/login",
                     "port": 8080
                   }
                 },
                 "livenessProbe": {
-                    "timeoutSeconds": 3,
+                    "timeoutSeconds": 240,
+		    "periodSeconds": 360,  
                     "initialDelaySeconds": 420,
                     "failureThreshold" : 30,
```

Comment:
with a polling interval of 6 mins, i think the failure threshold (number of failures required before we consider it dead) needs to be reduced.